File: src/test/java/com/polykhel/billnext/service/mapper/SubcategoryMapperTest.java
package com.polykhel.billnext.service.mapper;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

class SubcategoryMapperTest {

    private SubcategoryMapper subcategoryMapper;

    @BeforeEach
    public void setUp() {
        subcategoryMapper = new SubcategoryMapperImpl();
    }
}

File: src/test/kotlin/com/polykhel/billnext/service/mapper/SubcategoryMapperTest.kt
package com.polykhel.billnext.service.mapper

import org.junit.jupiter.api.BeforeEach

class SubcategoryMapperTest {

    private lateinit var subcategoryMapper: SubcategoryMapper

    @BeforeEach
    fun setUp() {
        subcategoryMapper = SubcategoryMapperImpl()
    }
}
